protostar/cli/protostar_arg_type.py

```diff
@@ -42,6 +49,15 @@ def parse_wei_arg_type(arg: str) -> Wei:
     return int(float(arg))
 
 
+def parse_input_arg_type(arg: str) -> InputType:
```

Comment:
edge case: `--inputs expression="x=1+1"`

Reply:
I don't think we should handle this at all. I think we should accept constants only as arguments. Going this way, we may want to handle evaluating external commands/code snippets that would eventually return the desired value. I don't think it makes sense.
Now, if you do `inputs x=1+1`, the cli will just throw an error and I'd stick with that.

Reply:
I meant passing string `x=1+1` or in general anything containing `=` or `+` or `\s` characters

Reply:
Yeah, I think this should make a shortstring I guess? But I'm don't know what data transformer is capable of.

Reply:
I tried to pass a string and it resulted in an error, only numbers are accepted I think. If you think we should be able to receive strings, that sounds like a separate issue, right?

Reply:
> ok makes sense, though I would still make sure that `parse_input_arg_type` is more robust. primarily, the `split` call should take into account possiblity of getting multiple `=` characters in input

@karol-bisztyga please address this

Reply:
Sorry, missed that!

Reply:
I think we can just throw in such a case [handle multiples = signs in inputs value](https://github.com/software-mansion/protostar/pull/1140/commits/09fa51cdf75eb614c353dba05d3e6e600870d24e). Looks like a situation that is not desired.